modules/index.d.ts

```diff
@@ -457,30 +457,30 @@ export type Fields = TypedMap<FieldOrGroup>;
 
 export type FieldItem = {
   items?: FieldItems, 
-  key: String, 
-  path?: String, 
-  label: String, 
-  fullLabel?: String, 
-  altLabel?: String, 
-  tooltip?: String,
-  disabled?: Boolean,
+  key: string, 
+  path?: string, 
+  label: string, 
+  fullLabel?: string, 
+  altLabel?: string, 
+  tooltip?: string,
+  disabled?: boolean,
 };
 type FieldItems = TypedMap<FieldItem>;
 
 export interface FieldProps {
   items: FieldItems,
-  setField(path: String): void,
-  selectedKey: String | Empty,
-  selectedKeys?: Array<String> | Empty,
-  selectedPath?: Array<String> | Empty,
-  selectedLabel?: String | Empty,
-  selectedAltLabel?: String | Empty,
-  selectedFullLabel?: String | Empty,
+  setField(path: string): void,
+  selectedKey: string | Empty,
+  selectedKeys?: Array<string> | Empty,
+  selectedPath?: Array<string> | Empty,
+  selectedLabel?: string | Empty,
+  selectedAltLabel?: string | Empty,
+  selectedFullLabel?: string | Empty,
   config?: Config,
   customProps?: {},
```

Comment:
Thank you so much for fixing all the capitalization in the types.  I can finally get rid of a lot of weird things like string interpolation!  

One other thing I've noticed while developing is the customProps being declared as a blank object is causing errors in my IDE around `<property-name> does not exist on type {}`.  Maybe I've got a type set up weird or wrong in the widgets but could you look at using something else?  `object` or `Record<any,any>`?